update.go

```diff
@@ -225,6 +243,34 @@ func nonConcurrentTestbootHandler(newRootPartition int) func(http.ResponseWriter
 	}
 }
 
+type ExtraUpdateHandler struct {
```

Comment:
Why is this type exported? It doesn’t need to be, right? Can you change it to `extraUpdateHandler` (lower-case e)?